src/ripple/app/ledger/impl/LedgerConsensusImp.cpp

```diff
@@ -230,48 +230,48 @@ LedgerConsensusImp::LedgerConsensusImp (
         LocalTxs& localtx,
         LedgerMaster& ledgerMaster,
         FeeVote& feeVote)
-    : app_ (app)
```

Comment:
How long does it take you to tell that all the operations are on member variables?
But the other arguments about reserved names are persuasive.